swiftgen-cli/ArgumentsUtils.swift

```diff
@@ -47,12 +47,20 @@ enum OutputDestination: ArgumentConvertible {
     }
   }
 
-  func write(content: String) {
+  func writeIfChanged(content: String) {
```

Comment:
I guess you're right. The helper function probably is of litte value.

I will remove it.